samples/notebooks/noise.ipynb

```diff
@@ -182,6 +182,69 @@
     "result = qsharp.run(\"Cat5()\", 1000, noise=(0.0, 0.1, 0.0))\n",
     "display(qsharp_widgets.Histogram(result))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulating with Qubit Loss\n",
+    "\n",
+    "For some qubit modalities, a qubit may undergo \"loss\" during execution. This means the physical system backing the qubit is unable to be measured or operated on. To test behaviors of an algorithm in the presence of qubit loss, you can use the `qubit_loss` parameter to `qsharp.run` and set the probability of a qubit being lost on each operation. Qubit loss is reported only at measurement time, where a special `Loss` value is returned that is neither `One` nor `Zero`. During simulation, a lost qubit is no longer acted on by gates and remains in the $\\ket{0}$ state until it is measured or reset, at which point the simulation reloads a fresh qubit to support future operations.\n",
+    "\n",
+    "In this example, we'll set the qubit loss probability to high 50% to ensure some qubits are lost during simulation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = qsharp.run(\"BellPair()\", 100, qubit_loss=0.5)\n",
+    "display(qsharp_widgets.Histogram(result))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the `Loss` value is not usable inside of the simulation, and any comparisons on that value will trigger a runtime failure. To avoid this failure, use `IsLossResult` to check whether the result value corresponds to qubit loss before using it in any branching logic:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "qsharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%qsharp\n",
+    "\n",
+    "operation CheckForLoss() : (Bool, Bool) {\n",
+    "    use q = Qubit();\n",
+    "    H(q);\n",
+    "    let res = MResetZ(q);\n",
```

Comment:
Was there a reason not to use `MResetZChecked` here? Or either works fine.

Reply:
Either works. I wanted to show the way to use an existing result from a "normal" measurement function.